build-scripts/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+
+if [[ "$BUILD_TYPE" == "android" ]];then
+    cd android
+    ./gradlew assembleRelease | tee
```

Comment:
Yup, you end up with "magnet/android/app/build/outputs/apk/*.apk" after this PR I'll get them pushed to Github (or some other storage).